app/tests/unit/routes/v1/status.spec.js

```diff
@@ -79,9 +158,24 @@ describe(`POST ${basePath}/:msgId`, () => {
   });
 
   it('should respond with a not found error', async () => {
-    const id = 'notfound';
+    const id = '00000000-0000-0000-0000-000000000000';
+    ChesService.prototype.getStatus.mockImplementation(() => {
+      throw new Problem(404);
+    });
+
     const response = await request(app).get(`${basePath}/${id}`);
+
     expect(response.statusCode).toBe(404);
+    expect(response.body.title).toMatch('Not Found');
   });
 
+  it('should respond with a validation error', async () => {
+    const id = 'badId';
+    const response = await request(app).get(`${basePath}/${id}`);
+
+    expect(response.statusCode).toBe(422);
+    expect(response.body).toBeTruthy();
+    expect(response.body.detail).toMatch('Validation failed');
+    expect(response.body.errors).toHaveLength(1);
```

Comment:
Test the actual validation error returned.

Reply:
Implemented